fs/accounting/stats.go

```diff
@@ -342,6 +337,19 @@ Elapsed time:  %10v
 	return buf.String()
 }
 
+// Wrapper around Fprintf to filer zero stat lines
+func addStatln(buf *bytes.Buffer, f string, a ...interface{}) {
```

Comment:
I think given that we need more complicated tests, I think I would inline this into `if x != 0 { _,_ = fmt.Printf("...%d...\n", x}`and just write it out 4 times rather than attempting to make it any more complicated

Reply:
Thanks, I see what you mean. Sorry for complicating. I will be back with fix.

Reply:
Give me a ping on the issue when you want another review :-)